edx_notifications/lib/tests/test_lib.py

```diff
@@ -208,6 +209,47 @@ def test_bulk_publish_notification_orm_query(self):
             self.assertEqual(len(notifications), 1)
             self.assertTrue(isinstance(notifications[0], UserNotification))
 
+    def test_mark_all_as_read(self):
+        """
+        Verify proper behavior when marking user notifications as read/unread
+        """
+        for i in range(10):
+            msg = NotificationMessage(
+                namespace='test-runner',
+                msg_type=self.msg_type,
+                payload={
+                    'foo': 'bar'
+                }
+            )
+            publish_notification_to_user(self.test_user_id, msg)
+
```

Comment:
It might be worth an assertion here about the number of unread messages, just to make sure this test isn't vacuous.